app/views/HonestyDeclarationPage.scala.html

```diff
@@ -34,13 +34,13 @@ <h1 class="govuk-heading-l">@messages("honestyDeclaration.heading")</h1>
     <p class="govuk-body">@messages("honestyDeclaration.p1")</p>
     <ul class="govuk-list govuk-list--bullet">
         <li>
-        @if(!reasonText.equals("")){
+        @if(!reasonText.equals("") && !isObligationAppeal){
             @messages("honestyDeclaration.li.1.1", reasonText)
         } else if(isObligationAppeal) {
             @messages("honestyDeclaration.obligation.li.1")
         }
 
-        @if(request.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Payment.toString)) {
+        @if(request.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Payment.toString) && !isObligationAppeal) {
             @messages("honestyDeclaration.lpp.li.1.2", friendlyDueDate)
         } else if(!isObligationAppeal) {
             @if(user.isAgent && didClientCauseLateSubmission){
```

Comment:
Looks like '!isObligationAppeal' boolean is used twice, can we use an ' if(!isObligationAppeal)'  at the top level to use this boolean once.

Reply:
Done